lib/backends/webgl/ops/matmul.ts

```diff
@@ -6,10 +6,20 @@ import {Tensor} from '../../../tensor';
 import {BroadcastUtil} from '../../../util';
 import {WebGLInferenceHandler} from '../inference-handler';
 import {ProgramInfo, RunData, WebGLOperator} from '../types';
+import {WebGLMatMulPacked} from './matmul-pack';
 
 export class WebGLMatMul extends MatMul implements WebGLOperator {
+  packedImpl: WebGLMatMulPacked;
```

Comment:
> packedImpl: WebGLMatMulPacked; [](start = 1, length = 31)

It's a bit weird to have an unpacked WebGLMatMul class contains a Packed MatMul object as member variable. Theoratically, those two implementations should be peer instead of one containing another. Maybe we can add a layer of abstraction, called WebGLMatMul, and rename the current unpacked implementation WebGLMatMulUnpacked (align with WebGLMatMulPacked). In WebGLMatMul, depending on inferenceHandler.session.pack, choose different kernel?

Reply:
fixed.